CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.9.0 (10th Nov 2020)
```

Comment:
I'm not sure what an appropriate version number would be... Consider `0.9.0` a placeholder for whatever seems appropriate.

Reply:
0.9.0 will be the next release number with web support, I consider this to be big new functionality but still missing a couple features for me to be happy to have Flutter Launcher Icons bumped to v1.0.0